docusaurus/docs/dev-docs/upgrade-tool.md

```diff
@@ -0,0 +1,194 @@
+
+Run the upgrade tool with the `codemods` parameter to execute a utility that allows selecting the codemods to be executed. With this command, only the codemods are run, the dependencies are not updated nor installed.
+
+```bash
+npx @strapi/upgrade codemods
+``` -->
+
+## Options
+
+The `npx @strapi/upgrade [major|minor|patch]` commands can accept the following options:
+
+| Option                                                                   | Description                                                      |
+| ------------------------------------------------------------------------ | ---------------------------------------------------------------- |
+| [`-n, --dry`](#simulate-the-upgrade-without-updating-any-files-dry-run)  | Simulate the upgrade without updating any files (default: false) |
+| [`-d, --debug`](#get-detailed-debugging-information)                     | Get more logs in debug mode (default: false)                     |
+| [`-s, --silent`](#execute-the-upgrade-silently)                          | Don't log anything (default: false)                              |
+| [`-p, --project-path <project-path>`](#select-a-path-for-the-strapi-application-folder) | Path to the Strapi project                                       |
+
+The following options can be run either with the `npx @strapi/upgrade` command alone or with the `npx @strapi/upgrade [major|minor|patch]` commands:
+
+| Option                                                                   | Description                                                      |
+| ------------------------------------------------------------------------ | ---------------------------------------------------------------- |
+| [`-V, --version`](#get-the-current-version)                       | Output the version number                                        |
+| [`-h, --help`](#get-help)                                                | Print command line options                                       |
+
+
+
+### Simulate the upgrade without updating any files (dry run)
+
+When passing the `-n` or `--dry` option, the codemods are executed without actually editing the files. The package.json will not be modified, and the dependencies will not be re-installed. Using this option allows simulating the upgrade of the codebase, checking the outcomes without applying any changes:
+
+Examples:
+
+```bash
+npx @strapi/upgrade major --dry
+npx @strapi/upgrade minor --dry
+npx @strapi/upgrade patch --dry
+```
+
+### Select a path for the Strapi application folder
+
+When passing the `-p` or `--project-path` option followed by a valid path you can specify in which folder the Strapi application is located.
+
+Example:
+
+```bash
+npx @strapi/upgrade major -p /path/to/the/Strapi/application/folder
+```
+
+### Get the current version
+
+When passing the `--version` option (or its `-V` shorthand), the current version of the upgrade tool is logged.
+
+Example:
+
+```sh
+$ npx @strapi/upgrade -V
+4.15.1
```

Comment:
I'm not sure the example output makes sense 🤔 I guess it would be more something like `1.0.1`? (i.e. the @strapi/upgrade version number is probably not aligned with the Strapi version number)

Reply:
Actually it is

To clarify, it could indeed not be related at some point + it doesn't required it be in sync, etc, but it happens to be :man_shrugging: